.github/workflows/Build.yaml

```diff
@@ -2,7 +2,7 @@ name: Build and Package
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, release ]
```

Comment:
Will the effect of having `release` in the branches clause cause everything to be built twice when the actual Release workflow is done?